themes/default/content/blog/esc-key-value-table-editor-launch/index.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "Introducing the Key-Value Table Editor for Pulumi ESC"
+date: 2024-03-26T00:00:00-07:00
+draft: false
+meta_desc: "The new enhancements to Pulumi ESC Editor streamlines the authoring experience of environments for developers"
+meta_image: "meta.png"
+authors:
+  - arun-loganathan
+  - kimberley-mackenzie
+tags:
+  - esc
+  - secrets
+  - config management
+---
+
+
+We're excited to announce the launch of the Key-Value Table Editor within Pulumi Environments, Secrets, and Configurations ([ESC](/product/esc)). At Pulumi, we're committed to providing flexible solutions that cater to diverse development needs and practices. This latest addition underscores our dedication to this principle by offering another user-friendly interface for managing complex configurations and secrets.
+
+<!--more-->
+
+The Key-Value Table Editor simplifies configuration tasks, allowing for direct input of plaintext values, and secrets. Key capabilities of the Table Editor include:
+
+- **CRUD Operations**: Perform Create, Read, Update, and Delete (CRUD) operations on your secrets and  plaintext values
+- **Decrypting Secrets**: Safely decrypt secrets one at a time, directly within the editor
+- **Environment Imports**: [Import environments](/docs/esc/environments/#importing-other-environments) and define the order of imports to suit your project's needs
+- **Variables Projections**: Make variables available as [Pulumi Config](/docs/esc/environments/#using-environments-with-pulumi-iac), [environment variables](/docs/esc/environments/#projecting-environment-variables), and files, streamlining your deployment and runtime configuration.
+- **Interpolations and References**: Add references to existing path-values, or add [interpolations](/docs/esc/environments/#interpolating-values) to dynamically generate values
+
+![Key-Value Table Editor](esc-key-value-table-editor.png)
+
+In the current release of the Table view, [provider configurations](/docs/esc/providers) and functions are set to read-only.  
+
+## What’s Next?
+
+Looking to the future, we are already planning the next set of enhancements for the Key-Value Table Editor. Soon, you'll be able to add and edit provider configurations directly within the table view, offering a more guided experience in utilizing Pulumi ESC's rich providers.
```

Comment:
I wonder if we can beef this section up a bit, talk to more detail on how we will provide a "guided" experience.  We want to give enough info to entice them to keep checking this space

Reply:
I initially planned on adding Hi-Fi designs in here but that isn't ready yet. 

What are you thinking in terms of what to add for 'how'

I could potentially see us adding an explainer for provider configuration as a way of reintroducing it for newcomers to ESC.

Reply:
I'm pro-reintroducing provider config! We probably could also take a sentence or two at the top to re-introduce ESC.